src/main/flight/pid.c

```diff
@@ -211,7 +211,7 @@ static void pidLuxFloat(pidProfile_t *pidProfile, controlRateConfig_t *controlRa
         if (lowThrottlePidReduction) RateError /= 4;
 
         // -----calculate P component
-        PTerm = RateError * pidProfile->P_f[axis] * PIDweight[axis] / 100;
+        PTerm = RateError * (pidProfile->P_f[axis]/4) * PIDweight[axis] / 100;
 
```

Comment:
For readability, Instead of modifying the 100 I just divide the terms. The compiler optimizes it so there's no need to do it here.